content/en/agent/basic_agent_usage/source.md

```diff
@@ -21,6 +21,31 @@ This page outlines the basic features of the Datadog Agent. If you haven't insta
 
 By default, your Agent is installed in its own sandbox at `~/.datadog-agent`. You're free to move this folder wherever you like. However, this article assumes that the Agent is installed in its default location, so be sure to modify the instructions accordingly if you decide to move them.
 
+## Linux Installation
+
+* Install [Go][2] version 1.11.5 or later. Remember to set your `GOPATH` and have `$GOPATH/bin` in your path.
+* Install Python version 3.x, along with development libraries.
+* Install CMake version 3.12+.
+* In your virtual environment, install Python dependencies. You may use `pip install -r requirements.txt`
+* Checkout the repo: `git clone https://github.com/DataDog/datadog-agent.git $GOPATH/src/github.com/DataDog/datadog-agent`
+* Install the project's dependencies with `invoke deps`. Ensure that `$GOPATH/bin` is in your `$PATH`.
```

Comment:
You can get rid of the `Ensure that GOPATH` line here, you've already verified it above.